### DISCUSSION TITLE
Thorium Hiding data fields

### DISCUSSION THREAD

**Discussion by vmsman** [Alex313031/thorium#317] [General]
First, thanks for this outstanding project.  I have found one web page that seems to be hiding a data field.  This site works in Google Chrome, Chromium and Brave.  Try these two sites:
`
https://www.foodnetwork.com/sponsored/sweepstakes/sink-your-teeth-sweepstakes-2023
https://www.cookingchanneltv.com/sponsored/sweepstakes/sink-your-teeth-sweepstakes-2023
`
There is an email field on the form and it is not showing up.  I turned off all blocking that I can find and still the field is not showing.  Any ideas?

**Comment by gz83** [comment]
The two sites you provided are not available on my end (region restrictions?), keep this question open to receive more responses

**Comment by gz83** [comment]
The new version no longer seems to have this issue.
